src/zenml/cli/user_management.py

```diff
@@ -32,40 +32,24 @@ def user() -> None:
 @user.command("get")
 def get_user() -> None:
     """Get the active user."""
-    cli_utils.print_active_profile()
-    cli_utils.declare(f"Active user: '{Repository().active_user_name}'")
-
-
-@user.command("set", help="Set the active user.")
-@click.argument("user_name", type=str)
```

Comment:
Why was this removed? I guess we should still support switching users locally, or how would that work now?